src/plugins/discover/tsconfig.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": "../../../tsconfig.base.json",
+  "compilerOptions": {
+    "composite": true,
+    "outDir": "./target/types",
+    "emitDeclarationOnly": true,
+    "declaration": true,
+    "declarationMap": true
+  },
+  "include": ["common/**/*", "public/**/*", "server/**/*", "../../../typings/**/*"],
```

Comment:
Could you move `fixtures` to the `discover` plugin? Ideally, all the files belong to a particular domain.

Reply:
Sounds like a good idea, I will do that on a separate PR (there's other Kibana app stuff to sort out)